cirq-core/cirq/devices/device.py

```diff
@@ -226,11 +228,14 @@ def nx_graph(self) -> Optional['nx.Graph']:
     def _value_equality_values_(self):
         graph_equality = None
         if self._nx_graph is not None:
-            graph_equality = (sorted(self._nx_graph.nodes()), sorted(self._nx_graph.edges()))
+            graph_equality = (
+                tuple(sorted(self._nx_graph.nodes())),
+                tuple(sorted(self._nx_graph.edges())),
+            )
 
         qubit_equality = None
         if self._qubits_set is not None:
-            qubit_equality = sorted(list(self._qubits_set))
+            qubit_equality = tuple(sorted(list(self._qubits_set)))
```

Comment:
We could directly compare the frozenset's as well?

Reply:
Right you are. Changed.